pages/Documents/ConversationalAI/Third-PartyBots/CustomEndpoint/advanced-features.md

```diff
@@ -395,3 +395,24 @@ can be seen in Figure 4.8:
 <img class="fancyimage" style="width:300px" src="img/customendpoint/customendpoint_richcontent_demo.gif">
 
 Figure 4.8 A Demo of sending raw response back from Custom Endpoint Service on receiving Rich Content Event
+
+### Receiving Last consumer message (Messaging Only)
+
+Third-Party bot now provides a way to add the last consumer message as a part of the welcome event (messaging only). When an ongoing conversation gets transferred to a new Agent or Skill, This enhancement will allow brands to respond to the last consumer message uttered as per their needs.
```

Comment:
you can remove `(messaging only)` from the description as it is in the heading already